README.md

```diff
@@ -18,40 +18,41 @@ import (
 )
 
 func main() {
-    ctx := context.TODO()
-    // create the client and connect to the cluster
-    client, err := hazelcast.StartNewClient(ctx) 
-    if err != nil {
-    	log.Fatal(err)
-    }
-    // get a map
-    people, err := client.GetMap(ctx, "people")
-    if err != nil {
-        log.Fatal(err)
-    }
-    personName := "Jane Doe"
-    // set a value in the map
-    if err = people.Set(ctx, personName, 30); err != nil {
-    	log.Fatal(err)
-    }
-    // get a value from the map
-    age, err := people.Get(ctx, personName)
-    if err != nil {
-        log.Fatal(err)
-    }
-    fmt.Printf("%s is %d years old.\n", personName, age)
-    // stop the client to release resources
-    client.Shutdown(ctx)
+	ctx := context.TODO()
+	// create the client and connect to the cluster on localhost
+	client, err := hazelcast.StartNewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// get a map
+	people, err := client.GetMap(ctx, "people")
+	if err != nil {
+		log.Fatal(err)
+	}
+	personName := "Jane Doe"
+	// set a value in the map
+	if err = people.Set(ctx, personName, 30); err != nil {
+		log.Fatal(err)
+	}
+	// get a value from the map
+	age, err := people.Get(ctx, personName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("%s is %d years old.\n", personName, age)
+	// stop the client to release resources
+	client.Shutdown(ctx)
 }
 ```
 
 ## Features
 
 * Distributed, partitioned and queryable in-memory key-value store implementation, called Map.
-* Additional data structures and simple messaging constructs such as Replicated Map, Queue, List, PNCounter, Set, Topic and others.
+* Additional data structures and simple messaging constructs such as Replicated Map, MultiMap, Queue, List, PNCounter, Set, Topic and others.
 * Support for serverless and traditional web service architectures with Unisocket and Smart operation modes.
 * Go context support for all distributed data structures.
 * Hazelcast Cloud integration.
+* SQL support (only on Hazelcast 5.x).
```

Comment:
Why don't we support 4.2 SQL?

Reply:
SQL was beta in 4.2, so we only support SQL for 5.0. We can talk about the decisions offline.